lib/src/usage.dart

```diff
@@ -250,14 +307,19 @@ class Usage {
   }
 }
 
-/// Pads [source] to [length] by adding spaces at the end.
-String padRight(String source, int length) {
-  final result = new StringBuffer();
-  result.write(source);
-
-  while (result.length < length) {
-    result.write(' ');
-  }
-
-  return result.toString();
+bool isWhitespace(String text, int index) {
```

Comment:
t'would be great to document from whence these magic values come

Reply:
Done.